app/main.py

```diff
@@ -1,6 +1,40 @@
+import os
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    api_key = os.environ.get("API_KEY")
+    filtering = "Paris"
+
+    weather_url = (
+        f"http://api.weatherapi.com/"
+        f"v1/current.json?key={api_key}&q={filtering}"
```

Comment:
Use `params` argument for `requests.get` instead of hardcoding query params to the url